.gitattributes

```diff
@@ -0,0 +1,5 @@
+# Don't allow people to merge changes to these generated files, because the result
```

Comment:
This was automatically generated by `rush init` (a new feature Pete recommended trying). I can revert if people prefer.